packages/lodestar/test/e2e/sync/reqResp.test.ts

```diff
@@ -44,8 +44,8 @@ block2.message.slot = BLOCK_SLOT + 1;
 describe("[sync] rpc", function () {
   this.timeout(20000);
   const sandbox = sinon.createSandbox();
-  const logger = new WinstonLogger({level: LogLevel.verbose});
-  logger.silent = true;
+  const logger = new WinstonLogger({level: LogLevel.debug});
+  logger.silent = false;
```

Comment:
If false, can this statement be deleted?

Reply:
I should probably put it back to true